augly/image/utils.py

```diff
@@ -279,3 +280,23 @@ def compute_stripe_mask(
     binary_mask = softmax_mask > (math.cos(math.pi * line_width) + 1) / 2
 
     return binary_mask
+
+
+def distort(
+        image: Image.Image,
+        distortion_type: str,
+        distortion_args: Tuple,
```

Comment:
```suggestion
        distortion_args: Tuple[float, float, float, float],
```

We can be more specific about the type here.

Reply:
Well, the reason I didn't specifically add the type hints like this is that with a dynamic length of tuple can be passed to `distort` function as distortion methods differ, so I left `Tuple` on purpose here.

Reply:
Ok, I see. It seems like for now it'll always be length 4 for the barrel & pincushion distortions, but if you're planning to potentially add more augmentations that have a different number of coeffs that seems fine!